src/config.rs

```diff
@@ -0,0 +1,86 @@
+use ansi_term::Color::Red;
+use ansi_term::Style;
+use atty::Stream::Stdout;
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+#[structopt(rename_all = "kebab-case")]
+pub struct Config {
```

Comment:
I'm not sure the name is ideal for our use case as we might start having "rua config" in the future. Also, it truly isn't a config -- it's CLI arguments.